### ISSUE TITLE
Upgrade OWL-RL dependency?

### ISSUE THREAD

**Issue by ceteri** [RDFLib/pySHACL#111]
Are there plans to bring the `OWL-RL` dependency up to v6.0+ so that these two libraries align with `RDFlib` v6.0.2+ ?
Currently this requires 5.2.3, which creates a `pip` conflict when loading all three libraries.
```
ERROR: Cannot install -r requirements.txt (line 16), owlrl>=6.0.2 and pyshacl==0.17.2 because these package versions have conflicting dependencies.                                                                                                                                 

The conflict is caused by:
    The user requested owlrl>=6.0.2
    pyshacl 0.17.2 depends on owlrl<6.0.0 and >=5.2.3
    The user requested owlrl>=6.0.2
    pyshacl 0.17.1 depends on owlrl<6.0.0 and >=5.2.3
```

**Comment by ashleysommer** [comment]
Hi @ceteri 
Thanks for bringing this to my attention. I thought this already works with OWL-RL 6.0, the dependency listed in pySHACL is `owlrl ^5.2.3,<7`, but there is a typo, it should be `>5.2.3,<7`.
I'll fix it, and push out a new release later today.

**Comment by ceteri** [comment]
Awesome, many thanks @ashleysommer 
and we really appreciate using `OWL-RL` !

**Comment by ashleysommer** [comment]
@ceteri A fix for this is in latest PySHACL v0.17.3

**Comment by ceteri** [comment]
thanks!
